utils.py

```diff
@@ -38,74 +36,31 @@ def flatten_custom_fields(elements: Sequence[Dict]) -> Dict:
 
 def load_orcid_information(orcids: Iterable[str]) -> Dict[str, Dict]:
     result = {}
-    orcid_api = OrcidAPI()
+    orcid_format_string = "https://orcid.org/{}/person.json"
     for orcid in orcids:
-        result[orcid] = orcid_api.personal_details(orcid_id=orcid)
+        response = requests.get(orcid_format_string.format(orcid))
+        if response.status_code == 200:
+            orcid_data = response.json()
+            result[orcid] = {
```

Comment:
Is it intentional to keep only the last match? If you assume there will be only one math, consider to ``break`` after storing the match.